### ISSUE TITLE
Home App Automations with Body Sensor intermittently working

### ISSUE THREAD

**Issue by magichew** [snOOrz/homebridge-aqara#72]
I have automations set up in the Home app using the Body Sensor and the Door/Window Sensor.

The Door/Window sensor reliably triggers the automations with every open and close of the door but the body sensor works intermittently.

I have notifications for the sensor set on my phone and there are times that I can stand in front of the sensor waving my arms and not trigger the sensor and so not trigger the Automation (turn Philips Hue bulb on).
There are also times that I can walk past a long way away from the sensor and trigger the sensor and receive a notification but the light doesn't come on.
Then there are other times that the sensor is triggered, I get the notification and the light comes on.

What I'm getting at is that it's seemingly random as to whether the body sensor is triggered or not and the Automation is triggered.

I've tried the sensor in various environment and with different bulbs (Philips Hue and Milight) with the same random results.

Anyone else finding this behaviour?

**Comment by adekka** [comment]
I have the same issue with the door/window sensors. I tried multiple setups and the behavior is unreliable.

**Comment by iCarl** [comment]
Are you guys having the same problem on Wifi and cellular?

**Comment by magichew** [comment]
I don't receive notifications when not on the local network (as per this thread I created... [)](https://github.com/snOOrz/homebridge-aqara/issues/68) but automations for the door/window sensor trigger (light's come on after sunset) without the notification reaching my phone.

**Comment by astrojetson8** [comment]
i think this has to do with how device polling is implemented.

**Comment by magichew** [comment]
It's strange. I just this second done it. Turn Hue light one via phone; it comes on. Turned it off via the phone; it turns off. Walk in the room; get notification of motion detected but light stays off.

**Comment by adekka** [comment]
> It's strange. I just this second done it. Turn Hue light one via phone; it comes on. Turned it off via the phone; it turns off. Walk in the room; get notification of motion detected but light stays off.

I recognize the same. Often (not always) the scene is triggerd to go on however the scene is not (always) triggered to go off.

**Comment by magichew** [comment]
Yes. Exactly the same here. It's just worked here this time but not several times before minutes earlier.

**Comment by adekka** [comment]
You'll see that this issue is a common issue for this plugin. Just hoping that someone can fix this on either this side or the homebridge side..

**Comment by magichew** [comment]
> You'll see that this issue is a common issue for this plugin. Just hoping that someone can fix this on either this side or the homebridge side.. btw, you've seen this one? https://github.com/Maxmudjon/homebridge-mijia

Page not found?

**Comment by adekka** [comment]
The link works for me. Or just look for homebridge-mijia

**Comment by magichew** [comment]
Strange. Link works now. What is this? A fork of this plugin? Have you used this? Is it not the same?
Perhaps the authors could collaborate?

**Comment by iCarl** [comment]
I recognize the user from previous issues in this plugin

**Comment by adekka** [comment]
I had the same questions :) I created an issue with those questions. Didn't start using it because I'm not sure wether it is an improvement.
